easybuild/easyconfigs/a/Arrow/Arrow-6.0.0-foss-2021a.eb

```
@@ -0,0 +1,59 @@
+easyblock = 'CMakeMake'
+
+name = 'Arrow'
+version = '6.0.0'
+
+homepage = 'https://arrow.apache.org'
+description = """Apache Arrow (incl. PyArrow Python bindings), a cross-language development platform
+ for in-memory data."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://archive.apache.org/dist/%(namelower)s/%(namelower)s-%(version)s']
+sources = ['apache-arrow-%(version)s.tar.gz']
+checksums = ['69d268f9e82d3ebef595ad1bdc83d4cb02b20c181946a68631f6645d7c1f7a90']
+
+builddependencies = [
+    ('CMake', '3.20.1'),
+    ('Autotools', '20210128'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.6'),
+    ('pkg-config', '0.29.2'),
+]
+
+# Arrow strongly prefers included jemalloc, so not including it as a dependency
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),  # for numpy
+    ('Boost', '1.76.0'),
+]
+
+separate_build_dir = True
+start_dir = 'cpp'
+
+# see https://arrow.apache.org/docs/python/development.html
+configopts = "-DARROW_PYTHON=on -DARROW_PARQUET=ON -DARROW_WITH_SNAPPY=ON "
+configopts += "-DCMAKE_INSTALL_LIBDIR=lib"
```

```suggestion
configopts += "-DCMAKE_INSTALL_LIBDIR=lib  -DPython3_ROOT_DIR=$EBROOTPYTHON"
```

The failure in https://github.com/easybuilders/easybuild-easyconfigs/pull/14259#issuecomment-955683680 is caused by Arrow deciding to use my EB venv Python and not the EB installed Python dependency. This fixes that by explicitly stating which Python to use.